CustomLinkedList/src/main/java/Node.java

```diff
@@ -0,0 +1,17 @@
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
+public class Node<V> {
```

Comment:
class is missing the toString(), equals(), and hashCode() methods, which are crucial for debugging and comparison.